### PR TITLE
ZK-4634: Classic themes don't handle toolbar overflowPopup and align=…

### PR DESCRIPTION
**Pull request by plijyfes**
…"end"

### CODE REVIEW

File: zul/src/archive/web/zul/less/_zkvariables.less [hunk 1]
```diff
@@ -225,9 +225,11 @@
 @groupbox3DContentPadding:     @containerPadding;
 
 // Toolbar
-@toolbarPadding:               6px 16px;
+@toolbarHorizontalPadding:     16px;
```
Comment (plijyfes) [inline zul/src/archive/web/zul/less/_zkvariables.less:228]: compact

File: zul/src/archive/web/js/zul/wgt/less/toolbar.less [hunk 2]
```diff
@@ -12,19 +12,30 @@
...
-			top: 50%;
-			.transform('translateY(-50%)');
-			right: 16px;
-
-			&:hover {
-				color: @colorPrimaryLighter;
+		&-on {
+			> .z-toolbar-overflowpopup-button {
+				cursor: pointer;
+				float: right;
```
Comment (plijyfes) [inline zul/src/archive/web/js/zul/wgt/less/toolbar.less:21]: remove float right

**Review by scribetw — CHANGES_REQUESTED** [review]
Iceblue compact, drop float: right

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**